### ISSUE TITLE
Merge PFAM model with Ensembl model, and import the unified model into MongoDB

### ISSUE THREAD

**Issue by onursumer** [genome-nexus/genome-nexus#89]
Current PFAM model is not very useful. Merge the PFAM model with Ensembl model and import a nested JSON structure into MongoDB.

**Comment by onursumer** [comment]
I think we can still keep the pfam collection in the database, but model it only with these 3 fields:

```JavaScript
pfamDomain: {
    pfamAccession: "" // a unique id (reffered as pfamA_acc in pfamA.txt)
    name: "" // short name (referred as pfamA_id in pfamA.txt)
    description: "" // a little longer description
}
```

Simply importing the `pfamA.txt` file as a standalone collection would do the trick. 
 Let's move it from `src/main/resources/pfamA.txt` to `data`. Also, the column headers in that file might be confusing. `pfamA_acc` is the actual unique id, and `pfamA_id` is the short name (that we display in the domain rectangles in cBioPortal)

We can also keep the pfam endpoint, but it will be a simple endpoint where we retrieve domains only by id (accession).

And then we can model the `transcript` this way:

```JavaScript
transcript {
    transcriptId: "" // a unique id
    ...
    ...
    domains : [{
        pfamAccession: "" // refers to the pfam collection
        start: 10
        end: 20
    }, {
    ...
    }]
}
```

This way we will still need 2 API calls in cBioPortal to get the domain data, but I think that shouldn't be a huge deal. @inodb what do you think?

**Comment by onursumer** [comment]
Fixed by #93 and #94
